doc/devguide/codebase/code-style.rst

```diff
@@ -159,7 +460,59 @@ TODO
 Macros
 ~~~~~~
 
-TODO
+Macro names are ALL_CAPS_WITH_UNDERSCORES.
```

Comment:
My interpretation of the Suricata usage.

Reply:
Yes, correct.